udata/api_fields.py

```diff
@@ -0,0 +1,100 @@
+from udata.api import api, fields
+from udata.models import db
+from bson import ObjectId
+import mongoengine
+
+def convert_db_to_field(key, field):
+    info = getattr(field, '__additional_field_info__', {})
+
+    params = {}
+    params['required'] = field.required
+
+    read_params = {}
+    write_params = {}
+
+    constructor = None
+    constructor_read = None
+    constructor_write = None
+
+    if info.get('convert_to'):
```

Comment:
In what case is this used?

Reply:
It was used in some of my test to allow specifying a custom `Field` type but the auto field type discovery is working good enough right now to not require it… I may remove it before merging if everything is working without.